### DISCUSSION TITLE
How to contribute ?

### DISCUSSION THREAD

**Discussion by julien040** [f1db/f1db#12] [Q&A] [answered]
Hi! First, I would like to say that I love your project!
Thank you for your incredible work!

But I think collecting all this data must be a lot of work.
Is there a guide to contribute to this repository and add some data?

Cheers,
Julien

**Comment by marceloverdijk** [comment, marked as answer]
Hi @julien040 ,

Thanks for reaching out and showing interest.
Yes, capturing, collecting, verifying, enhancing all data was quite a job. I spend years on this as some kind of hobby ;-)
Nowadays, it's not that much of work anymore; data is stable and it's mainly adding new after each race.

What kind of data would you like to contribute?

Cheers,
Marcel
